### DISCUSSION TITLE
12/26 안승민 블로그 3회차

### DISCUSSION THREAD

**Discussion by smahn9123** [Orm-camp/blog-feedback#55] [동글동글 4기 피드백]
https://comgu.tistory.com/entry/JavaScript-Browser-Object-ModelBOM
https://comgu.tistory.com/entry/Git-가장-최근-commit의-timestamp를-지금으로-변경하기

**Comment by YuSeungBeom** [comment]
몰랐던 팁 잘 정리해 주셔서 보기 편한 거 같습니다. 참고해서 잘 사용하겠습니다.

**Comment by Jungminchae** [comment]
유용한 정보인 것 같습니다. 다른 분들도 참고하셔서 익혀두시면 좋겠습니다!

**Comment by devSeyong** [comment]
꾸준히 글 작성하시는 모습이 멋지십니다. 항상 큰 동기부여가 됩니다!
